frontend/src/app/utils/srGraphQLErrorMessage.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { render, screen } from "@testing-library/react";
+
+import { getBody, getHeader } from "./srGraphQLErrorMessage";
+
+describe("getHeader", () => {
+  test("returns default header message if header not found", () => {
+    const message = "warning: results failed to upload; head: try again";
+    expect(getHeader(message)).toBe("Something went wrong");
+  });
+  test("returns header message up to semicolon", () => {
```

Comment:
Oh! Good point -- will add that! 

Also, I was going to first do a comma but we were using that in the error message so I chose a semicolon but now I'm wondering if we should split on a completely random character in case we do want to use a semicolon in the body? 🤔

Reply:
I like the semicolon because I think it's readable, and imo we shouldn't try to future-proof this too much. But it's up to you!

Reply:
Sounds good! I just added the note about formatting!